backend/entities/AddressType.cs

```diff
@@ -33,4 +33,13 @@ public PimsAddressUsageType(string id):this()
         }
         #endregion
     }
+
+    public static class AddressUsageTypes
+    {
+        public const string Mailing = "MAILADDR";
+        public const string ProofOfInsurance = "INSURE";
+        public const string RentalPayment = "RENTAL";
+        public const string PropertyNotification = "PROPNOTIFY";
+        public const string Physical = "PHYSADDR";
```

Comment:
This ones seem to be using the old codes

Reply:
Corrected, ugh.